drivers/led_strip/ws2812_gpio.c

```diff
@@ -140,7 +146,7 @@ static int send_buf(struct device *dev, uint8_t *buf, size_t len)
 
 	irq_unlock(key);
 
-	rc = clock_control_off(clk, CLOCK_CONTROL_NRF_SUBSYS_HF);
+	rc = onoff_release(mgr);
```

Comment:
This propagates to a behavior change to `led_strip_update_rgb()`: `clock_control_off` returns zero on success; `onoff_release()` may return a positive value on success.

Reply:
fixed